### PR TITLE
[test-only] Add grafana link to CI

### PR DESCRIPTION
**Pull request by ScharfViktor**
Added grafana dasboard link to CI

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -2884,6 +2884,14 @@ def k6LoadTests(ctx):
                     "sh %s/run_k6_tests.sh --ocis-log" % (dirs["base"]),
                 ],
             },
+            {
+                "name": "open-grafana-dashboard",
+                "image": OC_CI_ALPINE,
+                "environment": {},
+                "commands": [
+                    "echo `Grafana Dashboard: <https://grafana.k6.infra.owncloud.works>`_",
```
Comment (ScharfViktor) [inline .drone.star:2892]: Hopefully I'll see it as a link (not as text)

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text)https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text

Comment (saw-jan) [reply .drone.star:2892]: I doubt if it works. I got this error in my term
```bash
$ echo `Grafana Dashboard: <https://grafana.k6.infra.owncloud.works>`_
zsh: parse error near `>'
zsh: parse error in command substitution
```

Comment (ScharfViktor) [reply .drone.star]: left as text

File: .drone.star [hunk 2]
```diff
@@ -2884,6 +2884,14 @@ def k6LoadTests(ctx):
                     "sh %s/run_k6_tests.sh --ocis-log" % (dirs["base"]),
                 ],
             },
+            {
+                "name": "open-grafana-dashboard",
+                "image": OC_CI_ALPINE,
+                "environment": {},
```
Comment (saw-jan) [inline .drone.star:2890]:
```suggestion
```
others looks good :+1:

**Review by saw-jan — APPROVED** [review]

**Pull request merged**